src/components/Table/Table.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { BsFillPencilFill, BsFillTrashFill, BsCopy } from 'react-icons/bs';
+import { HttpClientService } from '../../HttpServerClient.ts';
+import './table.scss';
+
+interface Chef {
+    _id: string;
+    name: string;
+    description: string;
+    image: string;
+    // restaurants: string[];
+    status: string;
+}
+
+function Table() {
+    const [data, setData] = useState<Chef[] | null>(null);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const response: any = await HttpClientService.get('/chefs');
+                setData(response.data);
+            } catch (error) {
+                console.error('Error fetching data:', error);
+            }
+        }
+
+        fetchData();
+    }, []); // TODO: change it to redux
+
+    const keys = Object.keys(data ? data[0] : []);
+
+    return (
+        <div className='table-container'>
+            <table className='table'>
+                <thead>
+                    <tr>
+                        <th>ID</th>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Image</th>
+                        <th>status</th>
+                        <th></th>
+                    </tr>
```

Comment:
use Resources & remember that the column number can change depend the model values.